### PR TITLE
Dark Mode Feature

### LINKED ISSUE #205: Dark Mode for Status Site
Implement Dark Mode for Welcome Site.
Working Member - @abhinav1590 
Sample photos of Welcome site attached for reference 

Currently Welcome Site looks like this - 
image

After addition of Dark Mode it looks something like this 
image

### PR DESCRIPTION
**Pull request by abhinav1590**
Closes #205 

https://user-images.githubusercontent.com/86315672/140290064-bb52103d-c6df-4fbd-b806-16973069e073.mp4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "react-toastify": "^7.0.4",
     "sass": "^1.36.0",
     "sass-loader": "^12.1.0",
+    "styled-components": "^5.3.3",
```
Comment (harshith-venkatesh) [inline package.json:36]: Why did you install `styled-components` when we already have `sass`

File: src/components/Layout/index.tsx [hunk 2]
```diff
@@ -1,9 +1,13 @@
 import { FC, ReactNode } from 'react';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
+import { ThemeProvider } from 'styled-components';
```
Comment (harshith-venkatesh) [inline src/components/Layout/index.tsx:4]: Look for the way where you can build `ThemeProvider` without styled-component please

File: src/components/dark-theme/globalStyles.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { createGlobalStyle } from 'styled-components';
```
Comment (harshith-venkatesh) [inline src/components/dark-theme/globalStyles.js:1]: This can be handled without `styled-components`

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

**Comment by whyDontI** [comment]
Was stale for too long.

**Pull request closed without merging**